allennlp/modules/transformer/t5.py

```diff
@@ -56,14 +70,16 @@ def forward(self, hidden_states) -> FloatT:
         return hidden_states
 
 
+@T5FeedForwardProjection.register("gated-gelu")
 class T5DenseGatedGeluDense(TransformerModule, FromParams):
-    def __init__(self, hidden_size: int = 1024, ff_size: int = 4096, dropout: float = 0.1):
+    def __init__(self, hidden_size: int = 512, ff_size: int = 2048, dropout: float = 0.1):
         super().__init__()
-        self.wi_0 = nn.Linear(hidden_size, ff_size, bias=False)
-        self.wi_0.weight.data.normal_(mean=0.0, std=hidden_size ** -0.5)
-        self.wi_1 = nn.Linear(hidden_size, ff_size, bias=False)
-        self.wi_1.weight.data.normal_(mean=0.0, std=hidden_size ** -0.5)
-        self.wo = nn.Linear(ff_size, hidden_size, bias=False)
+        self.hidden_size = hidden_size
```

Comment:
Do we have to store the hidden size? Can't we just read it out of `self.wi_0` whenever we need it?

Reply:
Same question for `T5DenseReluDense`

Reply:
Good catch, didn't need this after all.